.github/workflows/smoke-test-create.yml

```
@@ -16,22 +16,24 @@ jobs:
     runs-on: ubuntu-latest
     env:
       UNITY_PATH: docker exec unity unity-editor
+      GITHUB_ACTOR: ${{ github.actor }}
+      UNITY_VERSION: ${{ inputs.unity-version }}
 
     steps:
     - name: Checkout
-      uses: actions/checkout@v3
+      uses: actions/checkout@f43a0e5ff2bd294095638e18286ca9a3d1956744 # v3
 
     - run: echo "::add-mask::${{ secrets.LICENSE_SERVER_URL }}"
 
     - name: Docker Login
       uses: docker/login-action@9780b0c442fbb1117ed29e0efdff1e18412f7567 # pinned v3
       with:
         registry: ghcr.io
-        username: ${{ github.actor }} 
-        password: ${{ secrets.GITHUB_TOKEN }} 
+        username: ${{ env.GITHUB_ACTOR }}
+        password: ${{ secrets.GITHUB_TOKEN }}
 
     - name: Start the Unity docker container
-      run: ./scripts/ci-docker.sh '${{ inputs.unity-version }}' 'base' '${{ secrets.UNITY_LICENSE_SERVER_CONFIG }}'
+      run: ./scripts/ci-docker.sh "${{ env.UNITY_VERSION }}" 'base' '${{ secrets.UNITY_LICENSE_SERVER_CONFIG }}'
```

```suggestion
      run: ./scripts/ci-docker.sh "${UNITY_VERSION}" 'base' '${{ secrets.UNITY_LICENSE_SERVER_CONFIG }}'
```
Probably, missed this one.